### PR TITLE
Simplify test runner

### PR DESCRIPTION
**Pull request by ZuluPro**
@benjaoming @mjs7231 What do you think about ?

I moved `runtests` to root and moved its settings part to `dbbackup.tests.settings`.
I also moved all the `tests` directory into `dbbackup/tests/` and `dbbackup/tests/testapp`

I improved functional tests by creating commands for count and populate database, and I test the results.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,3 +69,43 @@ We use `Travis`_ coupled with `Coveralls`_ as continious integration tools.
...
         :target: https://github.com/igrigorik/ga-beacon
+
+Tests
+=====
+
+Tests are stored in :mod:`dbbackup.tests` and for run them you must launch:
+
+::
+
+    python runtests.py
```
Comment (benjaoming) [inline README.rst]: That sounds cool.. didn't know that :) Recommend to amend the Makefile `test` target to reflect this also, such that `make test` works, too.

**Comment by benjaoming** [comment]
Agreeing with this, consolidate the Makefile, and this is a great improvement! Thanks for keeping the project alive and healthy! :)

**Comment by ZuluPro** [comment]
@benjaoming Makefile updated !

**Comment by benjaoming** [comment]
@ZuluPro do you have any idea about the new test failures? It seems they have started because of moving to a different way of calling the test runner?

**Comment by ZuluPro** [comment]
@benjaoming : From our code the problem come from moto: Mock library for AWS, so it isn't critical as it is supposed to disappear.

More precisely the problem come from Flask+Pypy3, I think since Flask 0.11 (since 5 days) Pypy3 isn't supported anymore. I tried to launch a test from DBBackup master, it failed: https://travis-ci.org/django-dbbackup/django-dbbackup/jobs/120767399

I'm going to add allowed failures on pypy3.

**Comment by ZuluPro** [comment]
@benjaoming I made it.
I think we should set an older version of Flask in pypy3 requirements (tox.ini).
Pypy3 uses Python 3.2.5 stdlib ([Ref](http://doc.pypy.org/en/latest/release-pypy3-2.3.1.html)), so it could be hard to maintain.

I have something to do, I'll ping you after and you check & merge ?

**Comment by ZuluPro** [comment]
@benjaoming I've done !
I fixed and added functional tests to CI, but it fails with postgres (failure allowed)

Could you review ?

**Comment by benjaoming** [comment]
Hurray, it passes!! :)

**Comment by benjaoming** [comment]
Not seeing anything that isn't an improvement!

**Comment by mjs7231** [comment]
:laughing:

**Pull request merged**